.github/workflows/publish_release.yml

```diff
@@ -21,7 +21,26 @@ jobs:
       - name: Install dependencies
         run: pip install -r requirements.txt
 
-      - name: Push documentaiton changes
+      - name: Push documentation changes
         uses: ./.github/actions/publish-docs-with-mike
         with:
           new_version: true
+
+  build-and-publish-wheel-to-pypi:
+    runs-on: ubuntu-latest
+    environment: "Publish Release"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Build wheel
+        uses: ./.github/actions/build-dist
+
+      - name: Publish distribution 📦 to PyPI
+        uses: pypa/gh-action-pypi-publish@v1.4.2
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Do you have a preference for not using Test PyPi to upload wheels during the dev process? One thought there is that it could catch upload issues (like the package classifiers that you fixed earlier, Pat) before attempting to upload a wheel to production PyPi.

I'm imagining a scenario where there's an upload issue that needs to be resolved with a code change & then we would need to manually delete and re-release the tag pointing to the newer commit

Reply:
[Block steps do not currently exist](https://github.community/t/blocking-job-or-step-for-approval-or-manual-continuation/122060), but are "on our roadmap to build".

[Manual triggers](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) could be used to run one off tests that upload to TestPyPi.